src/MultipartFormDataParser.php

```diff
@@ -192,7 +192,12 @@ public function parse($request)
                         } else {
                             fwrite($tmpResource, $value);
                             $fileConfig['tempFilename'] = $tmpFileName;
-                            $fileConfig['stream'] = new ResourceStream(['resource' => $tmpResource]); // save file resource, otherwise it will be deleted
+                            $fileConfig['stream'] = Yii::createObject([
```

Comment:
Why not simply: `new ResourceStream($tmpResource)` ?

Reply:
Update